test/Npgsql.Tests/ExceptionTests.cs

```diff
@@ -149,8 +149,10 @@ public void NpgsqlExceptionTransience()
         {
             Assert.True(new NpgsqlException("", new IOException()).IsTransient);
             Assert.True(new NpgsqlException("", new SocketException()).IsTransient);
+            Assert.True(new NpgsqlException("", new TimeoutException()).IsTransient);
             Assert.False(new NpgsqlException().IsTransient);
             Assert.False(new NpgsqlException("", new Exception("Inner Exception")).IsTransient);
+            Assert.False(new NpgsqlException("", new NotSupportedException()).IsTransient);
```

Comment:
Any reason to add this?

Reply:
I guess just another false check, to make sure it isn't returning true for the wrong case (NotSupportedException is a good case of something that should never be transient).

Reply:
That was my thought.

Reply:
Let's remove it since it covers only one of many exception types.

Reply:
done